e2e/cypress/integration/sinai_work_show_spec.js

```diff
@@ -17,4 +17,11 @@ describe('Sinai Work show pages', () => {
     cy.contains('div.title-row-show--siani', 'Sinai Syriac 70');
     cy.percySnapshot();
   });
+
+  it('IIIF Manifest Tooltip', () => {
+    cy.visit(Cypress.env('SINAI_BASE_URL') + '/catalog/' + encodeURIComponent('ark:/21198/z1s76kq5'));
+    cy.get('.si-link-iiif-manifest').trigger('mouseover');
+    cy.get('.si-link-iiif-manifest').should('be.visible');
```

Comment:
instead of this line test for 
cy.contains('div.tooltip-inner','IIIF Manifest');

Reply:
made this change